### PR TITLE
add example

### PR DESCRIPTION
**Pull request by mishin**
I try
to make suitable example
also help me get list of friend in script get_friends.pl

### CODE REVIEW

File: ex/get_friends.pl [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+say Dumper $token_ref->{access_token};
+
+# user authorization
+my $fb = Facebook::OpenGraph->new(
+    +{
+        app_id       => 35324055986753,
+        secret       => '291d7f26f909c137bba22cdf809c393',
+        namespace    => 'mishin_narod_ru',
+        redirect_uri => 'https://sample.com/auth_callback',
+    }
```
Comment (oklahomer) [inline ex/get_friends.pl]: Are these your actual app settings?
If so, I believe you shouldn't include them here.
If not, maybe it's a good idea to change this to obvious dummy data or environment variable such as below so you won't surprise anyone:
```perl
Facebook::OPenGraph->new(+{ app_id => $ENV{YOUR_APP_ID, ....});
```

File: ex/get_user_info.pl [hunk 2]
```diff
@@ -0,0 +1,9 @@
+use Modern::Perl;
+use Facebook::OpenGraph;
+
+# fetching public information about given objects
```
Comment (oklahomer) [inline ex/get_user_info.pl]: Maybe it's not a good idea to include an example to fetch public information without access_token.
After v2.0, we will need access_token to fetch those public information, too.
And v1.0 is going to be deprecated shortly.
I was going to remove this kind of synopsis on next version.

**Comment by oklahomer** [comment]
Thanks for the example codes, @mishin.
I'm sure tangible examples help others understand the basics, and I really appreciate your work.
I added some comments to your codes in case you just might want to consider some additional changes.

Thank you.

File: ex/get_friends.pl (hunk 1 above)
Comment (mishin) [inline ex/get_friends.pl]: Hi, Oklahomer,
it's not real , I change it

**Comment by oklahomer** [comment]
Closing since it no longer seems to be active.

**Comment by mishin** [comment]
yes, true

**Pull request closed without merging**